src/middlewares/rate-limiter.ts

```diff
@@ -0,0 +1,13 @@
+import rateLimiter from "express-rate-limit";
+
+export const limiter = rateLimiter({
+  max: 120,
```

Comment:
The limit of 120 per 5 seconds is too big and doesn't help. Limit it to 20 requests per 60 seconds.